src/lib.rs

```diff
@@ -591,7 +670,7 @@ mod test {
 		let variables: &dyn VariableMap<Value = &String> = &variables;
 
 		let_assert!(Ok(expanded) = substitute("one ${aap}", variables));
-		assert!(expanded == "one noot");
+		assert_eq!(expanded, "one noot");
```

Comment:
Same here, don't change this. [`assert2::assert!()`](https://docs.rs/assert2/latest/assert2/macro.assert.html) is better.